Makefile

```diff
@@ -1,14 +1,17 @@
 # Don't edit Makefile! Use conf-* for configuration.
 
 SHELL=/bin/sh
+NROFF=nroff
 
 default: it
 
 .PHONY: check clean default it man
 
+%.0:
+	$(NROFF) -man $^ > $@
```

Comment:
While I prefer this syntax, this is not POSIX, so will require gmake on BSDs (which have their different extensions).

Same goes for `$^`.

If we want an automatic make rule with a variable, we can go with this syntax which is portable (tested with OpenBSD make and GNU make):

```
.SUFFIXES: .0 .1 .3 .5 .7 .8

.1.0:
	$(ROFF) $< >$@
```